lib/realtime/helpers.ex

```diff
@@ -321,7 +323,10 @@ defmodule Realtime.Helpers do
           case elem(state, 11) do
             {pid, _, Postgrex.Protocol, opts} ->
               if opts[:hostname] == "db.#{tenant_id}.supabase.co" do
-                IO.puts("Killing #{inspect(pid)}")
+                Logger.warn(
+                  "Killing connection " <> inspect(pid) <> "with reason " <> inspect(reason)
+                )
+
```

Comment:
- there will not be any output if you invoke this function from the REPL
- `warm` is deprecated

Reply:
I keep forgetting it ... 

and I just want to check it in logflare since we will need to trigger this to all nodes (should actually do that erpc call here 🤔 )